### PR TITLE
Feature/pypi

### PR DESCRIPTION
**Pull request by Dedongala**

### CODE REVIEW

File: scripts/publish-pypi.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  const args = parser.parseArgs();
+
+  try {
+    const branches = await getNewlyCreatedBranches();
+    console.log('Branches:', branches);
+
+    const latestBranch = branches[0];
+    console.log('Latest Branch:', latestBranch);
+
+    const answer = await question(chalk.cyan.bold(`Is this the right branch '${latestBranch}' you are looking for? (y/n): `));
```
Comment (mattsoulanille) [inline scripts/publish-pypi.ts:42]: Nit: If the default is `no`, then it's good to indicate it with a capital letter: `(y/N)`.

File: scripts/publish-pypi.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      console.log(chalk.blue.bold('Current directory:', shell.pwd().toString()));
+      const executeCommand = (code, stdout, stderr) => {
+        if (code !== 0) {
+          console.error(`Error executing the command: ${stderr}`);
+          return;
+        }
+        console.log(stdout);
+      };
+      shell.exec('bazel clean', executeCommand);
+      shell.exec('bazel build python3_wheel', executeCommand);
```
Comment (mattsoulanille) [inline scripts/publish-pypi.ts:57]: You might be able to use [the `$` function we have in release-util.ts](https://github.com/tensorflow/tfjs/blob/master/scripts/release-util.ts#L236) instead.

**Review by mattsoulanille — APPROVED** [review]
Looks good! I have just a few changes.

**Review by pyu10055 — APPROVED** [review]
thanks!

**Pull request merged**